### PR TITLE
Add config files for mainnet

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add config files for mainnet
- Update deploy scripts for mainnet
- Fix some bugs in deploy scripts and genesis block generation config

### CODE REVIEW

File: blockchain-configs/mainnet-prod/blockchain_params.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  "_description": "Blockchain params for the mainnet-prod on cloud.",
+  "token": {
+    "name": "AI Network",
+    "symbol": "AIN",
+    "total_supply": 700000000,
+    "bridge": {
+      "ETH": {
+        "3": {
+          "0xB16c0C80a81f73204d454426fC413CAe455525A7": {
+            "token_pool": "0x20ADd3d38405ebA6338CB9e57a0510DEB8f8e000",
```
Comment (liayoo) [inline blockchain-configs/mainnet-prod/blockchain_params.json:11]: Oh, I forgot about this.. let me create an account for the pool & update this address

File: blockchain-configs/mainnet-prod/blockchain_params.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
+  "_description": "Blockchain params for the mainnet-prod on cloud.",
+  "token": {
+    "name": "AI Network",
+    "symbol": "AIN",
+    "total_supply": 700000000,
+    "bridge": {
+      "ETH": {
+        "3": {
+          "0xB16c0C80a81f73204d454426fC413CAe455525A7": {
```
Comment (liayoo) [inline blockchain-configs/mainnet-prod/blockchain_params.json:10]: Could you update bridge/ETH/3/0xB16c0C80a81f73204d454426fC413CAe455525A7 to bridge/ETH/1/0x3A810ff7211b40c4fA76205a14efe161615d0385 ?

File: blockchain-configs/mainnet-prod/blockchain_params.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    "apps_bandwidth_budget_ratio": 0.495,
+    "free_bandwidth_budget_ratio": 0.005,
+    "min_staking_for_app_tx": 0,
+    "min_balance_for_service_tx": 0,
+    "max_function_urls_per_developer": 3,
+    "default_developers_url_whitelist": {
+      "0": "https://*.ainetwork.ai",
+      "1": "https://*.ainize.ai",
+      "2": "https://*.afan.ai",
+      "3": "http://localhost:3000"
```
Comment (liayoo) [inline blockchain-configs/mainnet-prod/blockchain_params.json:93]: Should we drop the localhost?

File: blockchain-configs/mainnet-prod/node_params.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  "CHAINS_N2B_MAX_NUM_FILES": 100000,
+  "CONSOLE_LOG": false,
+  "CORS_WHITELIST": [
+    "https://ainetwork\\.ai",
+    "https://ainize\\.ai",
+    "https://afan\\.ai",
+    "\\.ainetwork\\.ai$",
+    "\\.ainize\\.ai$",
+    "\\.afan\\.ai$",
+    "http://localhost:3000"
```
Comment (liayoo) [inline blockchain-configs/mainnet-prod/node_params.json:15]: ditto

File: blockchain-configs/mainnet-prod/node_params.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  "REST_FUNCTION_CALL_TIMEOUT_MS": 10000,
+  "REQUEST_BODY_SIZE_LIMIT": "100mb",
+  "SNAPSHOTS_INTERVAL_BLOCK_NUMBER": 1000,
+  "SYNC_MODE": "full",
+  "TARGET_NUM_OUTBOUND_CONNECTION": 3,
+  "TRACKER_UPDATE_JSON_RPC_URL": "http://34.81.167.141:8080/json-rpc",
+  "TRACKER_UPDATE_INTERVAL_MS": 15000,
+  "TRAFFIC_DB_INTERVAL_MS": 60000,
+  "TRAFFIC_DB_MAX_INTERVALS": 60,
+  "TRAFFIC_STATS_PERIOD_SECS_LIST": {
```
Comment (liayoo) [inline blockchain-configs/mainnet-prod/node_params.json:58]: question) are we keeping all 3?

File: start_node_incremental_gcp.sh [hunk 6]
```diff
@@ -54,7 +54,7 @@ if ! [[ $3 =~ $number_re ]] ; then
     printf "Invalid <Node Index> argument: $3\n"
     exit
 fi
-if [[ "$3" -lt 0 ]] || [[ "$3" -gt 6 ]]; then
+if [[ "$3" -lt 0 ]] || [[ "$3" -gt 9 ]]; then
```
Comment (liayoo) [inline start_node_incremental_gcp.sh:57]: 👍

File: blockchain-configs/mainnet-prod/blockchain_params.json (hunk 1 above)
Comment (platfowner) [reply blockchain-configs/mainnet-prod/blockchain_params.json:11]: thanks for handling it!

File: blockchain-configs/mainnet-prod/node_params.json (hunk 5 above)
Comment (platfowner) [reply blockchain-configs/mainnet-prod/node_params.json:57]: as discussed offline, let's just keeps it~

File: blockchain-configs/mainnet-prod/blockchain_params.json (hunk 2 above)
Comment (platfowner) [reply blockchain-configs/mainnet-prod/blockchain_params.json]: done, thanks!

File: blockchain-configs/mainnet-prod/blockchain_params.json (hunk 3 above)
Comment (platfowner) [reply blockchain-configs/mainnet-prod/blockchain_params.json]: dropped, thanks!

File: blockchain-configs/mainnet-prod/node_params.json (hunk 4 above)
Comment (platfowner) [reply blockchain-configs/mainnet-prod/node_params.json]: ditto~

**Comment by platfowner** [comment]
All comments addressed, ptal~ @liayoo

File: start_node_genesis_gcp.sh [hunk 7]
```diff
@@ -150,14 +151,14 @@ fi
...
 
 KEYSTORE_DIR=testnet_dev_staging_keys
-if [[ $SEASON = 'spring' ]]; then
-    export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/testnet-prod
-    export TRACKER_UPDATE_JSON_RPC_URL=http://35.221.137.80:8080/json-rpc
-    export PEER_CANDIDATE_JSON_RPC_URL="http://35.221.184.48:8080/json-rpc"
+if [[ $SEASON = 'mainnet' ]]; then
+    export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/mainnet-prod
+    export TRACKER_UPDATE_JSON_RPC_URL=http://34.81.167.141:8080/json-rpc
+    export PEER_CANDIDATE_JSON_RPC_URL="http://104.199.237.250:8080/json-rpc"
```
Comment (minsulee2) [inline start_node_genesis_gcp.sh:157]: Hope no abusing for this :)

Comment (platfowner) [reply start_node_genesis_gcp.sh:157]: i hope so too. 🙏

**Review by liayoo — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**